### PR TITLE
Navigation [YEP-76]

### PR DESCRIPTION
**Pull request by g1039**

### CODE REVIEW

**Comment by g1039** [comment]
@smn @bruskiza @miltontony Ready for review

File: tuneme/templates/core/tags/section_listing_homepage.html [hunk 1]
```diff
@@ -26,7 +26,7 @@ <h2 class="list-header">{% trans "Latest" %}</h2>
           <p>{{page.subtitle}}</p>
           {% get_comment_count for page as comment_count %}
           {% if comment_count > 0 %}
-            <div class="comments-count"><a href="#">{{comment_count}} <span>{% trans "comment" %}{{comment_count|pluralize}}</span></a></div>
+            <div class="comments-count"><a href="{% pageurl page %}">{{comment_count}} <span>{% trans "comment" %}{{comment_count|pluralize}}</span></a></div>
```
Comment (miltontony) [inline tuneme/templates/core/tags/section_listing_homepage.html]: @TrevorOctober a suggestion here to link it directly to the comments section. It'll need 2 things
1. add an `id` to the article page where the [comments section starts](https://github.com/praekelt/molo-tuneme/blob/feature/issue-73-yep-76/tuneme/templates/core/article_page.html#L88) You can add the following div around all the comments `<div id="comments-list">`
2. change this page url to the following `<a href="{% pageurl page %}#comments-list">` which links users directly to the element with the id `comments-list`

**Comment by g1039** [comment]
@miltontony Ready for review

**Comment by bruskiza** [comment]
looks good to me... let 'ol hawkeye milton review.

File: tuneme/templates/core/article_page.html [hunk 2]
```diff
@@ -84,9 +84,11 @@
...
   </div>
 </div>
+<div id="comments-list">
 {% get_molo_comments for self as comment_list %}
 {% for node in comment_list|slice:":5" %}
   {% include "comments/comment.html" %}
 {% endfor %}
 
 {% endblock %}
+</div>
```
Comment (miltontony) [inline tuneme/templates/core/article_page.html:92]: This needs to be before the {% endblock %}

**Comment by miltontony** [comment]
otherwise :+1: once the div tag is fixed

**Comment by g1039** [comment]
@bruskiza @miltontony ready for re-review

**Pull request merged**